### PR TITLE
Test 'test262' test suite on Travis CI

### PR DESCRIPTION
**Pull request by LaszloLango**
JerryScript-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

**Comment by LaszloLango** [comment]
Note: bestPractise and int402 tests are excluded and it only tests release builds.

File: tools/runners/run-test-suite-test262.sh [hunk 1]
```diff
@@ -28,91 +27,44 @@ then
...
-  "${ARG_TEST262_PATH}"/tools/packaging/test262.py --command "timeout ${TIMEOUT} ${ARG_ENGINE}"                        \
-                                                   --tests="${ARG_TEST262_PATH}" --full-summary "${ARG_CHAPTER}"       \
-                                                   > "${REPORT_PREFIX}"."${ARG_CHAPTER}"
-}
+rm -rf "${PATH_TO_TEST262}/test/suite/bestPractice"
+rm -rf "${PATH_TO_TEST262}/test/suite/intl402"
 
-function show_report_results () {
-  ARG_CHAPTER="$1"
+echo "Starting test262 testing for ${ENGINE}. This could take awhile. Please be patient."
```
Comment (zherczeg) [inline tools/runners/run-test-suite-test262.sh]: :D

Be less personal. E.g: Running test262 may take a several minutes.

**Comment by zherczeg** [comment]
LGTM

**Comment by tilmannOSG** [comment]
@LaszloLango Out of curiosity, how long does it actually take to run in the cloud with Travis CI?

**Comment by LaszloLango** [comment]
@tilmannOSG, https://travis-ci.org/Samsung/jerryscript/builds/176977236
approximately 3,5 - 4 minutes

**Pull request merged**